helm/observability-bundle/values.yaml

```diff
@@ -49,6 +49,18 @@ apps:
                 # Add app label.
                 - targetLabel: app
                   replacement: kubernetes
+                metricRelabelings:
+                 # provided with original servicemonitor
+                - action: drop
+                  regex: apiserver_request_duration_seconds_bucket;(0.15|0.2|0.3|0.35|0.4|0.45|0.6|0.7|0.8|0.9|1.25|1.5|1.75|2|3|3.5|4|4.5|6|7|8|9|15|25|40|50)
```

Comment:
Do we really need to list all the buckets here ? Can we just drop the metric using its name and skipping the `le` label ?

Reply:
Phoenix is using apiserver_request_duration_seconds_count in their SLO but not the bucket it seems.

Reply:
> Do we really need to list all the buckets here ? Can we just drop the metric using its name and skipping the `le` label ?

This part just keeps the default settings.
It drops only the selected buckets, so we still have a histogram but with lower definition.

Reply:
@TheoBrigitte as discussed, I improved my comment in the code.
Do you like it better this way?

Reply:
Yes, thanks.